xbmc/video/ContextMenus.cpp

```diff
@@ -164,7 +237,12 @@ bool CPlay::IsVisible(const CFileItem& itemIn) const
     return false;
 
   if (item.m_bIsFolder)
-    return false; //! @todo implement
+  {
+    if (StringUtils::StartsWith(item.GetPath(), "pvr://recordings/"))
+      return true;
+    else
+      return false; //! @todo implement
```

Comment:
What should be implemented here?

Reply:
Once video and music components are changing to the "new" context menu system, the respective implementation (which are now located in the resp. window classes) need to go here.

Reply:
Ah, ok. Makes sense.